steps/2/README.md

```diff
@@ -115,7 +115,7 @@ and contains in `entries` the "variable" `dot`, linked to the respective chain a
 
 > Note: the command `bunx papi add dot -n polkadot` is same if we used an rpc endpoint with the flag -w, meaning:
 >
-> `$ bunx papi add -w wss://polkadot-collectives-rpc.polkadot.io dot`
+> `$ bunx papi add dot -w wss://polkadot-collectives-rpc.polkadot.io`
```

Comment:
why `dot` not `polkadot`?

Reply:
`dot` is just a "variable" name, that we assigned the metadata. This is saved as information in [the polkadot-api.json config file](https://github.com/wirednkod/papi-intro-tutorial/blob/gitorial/.papi/polkadot-api.json#L5). It is used for having access to later in the app (see [here](https://github.com/wirednkod/papi-intro-tutorial/blob/gitorial/step1.ts#L1)). That "variable" could be named "nikos" or "shawn" or whatever. I chose to use `dot` as the name to keep the correllation of the `polkadot`'s metadata but at the same time not to confuse the reader.
All of the above are mentioned along the tutorial